.storybook/config.js

```diff
@@ -1,10 +1,14 @@
-import 'font-awesome/css/font-awesome.min.css'
 import 'tachyons-egghead'
 import {configure, setAddon} from '@kadira/storybook'
-import infoAddon from '@kadira/react-storybook-addon-info';
+import infoAddon from 'react-storybook-addon-info';
```

Comment:
I forked this to make it look better and remove uneeded pieces. Eventually we can replace storybook with a custom solution, but this is pretty minimal for now and works better than before.